app/views/index.erb

```diff
@@ -0,0 +1,35 @@
+<body>
+  <br>
+  <h1>Welcome to Chitter</h1>
+  <br>
+  <br>
+  <form class="form-inline" method='post' action='/login'>
+    <legend>Please login here</legend>
+    <div class="form-group">
+      <label for="inputUsername">Username: </label>
```

Comment:
the `for` attribute should be the `id` for the related `input` tag.  In this case, the `input` tag that follows should have `id="inputUsername"`